lib/main_common.pm

```diff
@@ -398,6 +398,7 @@ sub load_reboot_tests {
             }
         }
         loadtest "installation/first_boot";
+        loadtest "installation/lvm_thin_check" if get_var('LVM_THIN_LV');
```

Comment:
IMHO, we should move this to console tests, as running `consoletest_setup` is definitely not a bad idea. I also don't find it wrong if we run only this test module after we finish installation, but it doesn't belong here. WDYT?

Reply:
No probs at all. This is just a mere suggestion where it can be placed, besides it saved a bit waiting time for me.

Reply:
hm...but on the other thought, we would need to run console tests after installation, which does not happen always.

Reply:
Yep, so basically I would say that it would be enough just to run this test module (with consoletest_setup in front). Or put it in console tests and execute same set of tests we run on cryptlvm (e.g. https://openqa.suse.de/tests/1896209).